sass/layout/_navbar.scss

```diff
@@ -12,10 +12,11 @@ body {
 
 .nav {
   z-index: 100;
-  height: 50px;
+  height: 5rem;
   width: 100%;
+  position: fixed;
```

Comment:
Have you had the chance to look into `sticky` option?  
With fixed you probably need to add margin/padding to the content itself to account for the navigation being `fixed`. 🤔